### PR TITLE
Issue four Closes #5

### LINKED ISSUE #5: Styling: Master Detail View
This is a wireframe—not a comp. You can choose to style it however you'd like. You may choose to prioritize other issues over this one for now.

Users should see the list of reminders along the left side and the individual new note on the right.

remember layout

### PR DESCRIPTION
**Pull request by kccrs**
## Purpose

Users should see the list of reminders along the left side and the individual new note on the right.

## Approach

I used SASS for styling and Adobe Color to identify a color scheme.

### Learning

I used my existing Sass knowledge and searched Ember docs and the Google for how to reformat the date using moment.js.  Also Casey Metz helped me figure that one out.

### Test coverage 

I did not test the styling. 

### Follow-up tasks

- I will work on reverting the edit fields.

#### After

rememberstyle

### CODE REVIEW

File: app/transforms/date.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import DS from 'ember-data';
+import moment from 'moment';
+
+export default DS.Transform.extend({
+ deserialize(serialized) {
+   if (serialized){return moment(serialized).format("dddd, MMM Do, YYYY");}
```
Comment (brittanystoroz) [inline app/transforms/date.js]: Would be nice if this code could breathe a little bit, separated out onto an extra line

File: app/transforms/date.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+   return serialized;
+ },
+
+ serialize(deserialized){
+   if(deserialized){return moment(deserialized).toISOString();}
```
Comment (brittanystoroz) [inline app/transforms/date.js]: Same here.

Comment (kccrs) [inline app/transforms/date.js]: Got it.

File: app/transforms/date.js (hunk 1 above)
Comment (kccrs) [inline app/transforms/date.js]: Done.

**Comment by brittanystoroz** [comment]
Cute app :)

**Pull request merged**